frigate/object_processing.py

```diff
@@ -165,6 +166,9 @@ def update(self, current_frame_time, obj_data):
         if self.computed_score > self.top_score:
             self.top_score = self.computed_score
         self.false_positive = self._is_false_positive()
+        # Hold previous active state for checking further down.
+        previously_active = self.active
```

Comment:
I don't think it makes sense to use a separate variable for this, it can be stored as part of the previous object data, right?